esx_service/cli/vmdkops_admin.py

```diff
@@ -1019,6 +1022,10 @@ def generate_tenant_access_ls_rows(privileges):
             datastore = ""
         else:
             datastore = vmdk_utils.get_datastore_name(p.datastore_url)
+
+        if datastore is None:
+            return rows
```

Comment:
Same. This can happen for one privilege. Need to appropriately consider the list of privileges.